devel/example_notebooks/example1.ipynb

```diff
@@ -79,7 +88,10 @@
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "# Net yet working - need to implement selection\n",
+    "# sorting_views.Curation(sorting=sorting, recording=recording)"
```

Comment:
The curation jupyterlab widget is not quite working yet because we need to implement selection